scripts/install-windows-blas-lapack.ps1

```
@@ -0,0 +1,200 @@
+    } else {
+      $kept.Add($trimmed)
+    }
+  }
+
+  Remove-Item Env:FC -ErrorAction SilentlyContinue
+  Remove-Item Env:F77 -ErrorAction SilentlyContinue
+  Remove-Item Env:F90 -ErrorAction SilentlyContinue
+  $env:Path = [string]::Join(";", $kept)
+}
+
+function Assert-InstalledArtifacts {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Assert-InstalledArtifacts' uses a plural noun. A singular noun should be used instead.

Suggested fix: Singularized correction of 'Assert-InstalledArtifacts'

(PSUseSingularNouns)